Sprint-1/exercises/random.js

```diff
@@ -3,6 +3,9 @@ const maximum = 100;
 
 const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 
+
+console.log(`Num is a random number between ${minimum} and ${maximum}: ${num}`);
+
```

Comment:
Including or excluding 1 and 100?

To test your understanding, how would you write an expression (without using any variable in the expression) that yields a random integer between -5 and 3 (including -5 and 3)?